packages/run-v5/test/colorize.test.js

```diff
@@ -6,10 +6,34 @@ describe('colorize', () => {
     console.log(colorize(`2018-01-01T00:00:00.00+00:00 heroku[${type}]: ${input}`))
   }
 
+  const testColorizeRedis = (type, input) => {
```

Comment:
That's a good question. Initially yes, but now looking back, I think this could have just been called within the `it` block. When we come back to clean up this file, I'll update that test. 👍